modules/commands/textgen_webui.py

```diff
@@ -44,6 +47,8 @@ def handle_custom_chat(logline: LogLine, shared_dict: dict):
     )
 
     message = add_prompts_by_flags(logline.prompt)
+    message = message.replace("!chat", "")
```

Comment:
That works, but in the app, another approach is used. This would fit better:
`message = message.removeprefix(config.CUSTOM_MODEL_CHAT_COMMAND).strip()`

Reply:
Sorry, I was a bit sleepy when I coded this, so I didn't account for the fact that if the command is changed in config.ini, it won't work.